chill-scala/src/main/scala/com/twitter/chill/ScalaKryoInstantiator.scala

```diff
@@ -198,8 +199,16 @@ class AllScalaRegistrar extends IKryoRegistrar {
       .forSubclass[scala.Enumeration#Value](new EnumerationSerializer)
 
     // use the singleton serializer for boxed Unit
-    val boxedUnit = scala.Unit.box(())
+    val boxedUnit = scala.runtime.BoxedUnit.UNIT
```

Comment:
this won't be necessary in M5, fixed in https://github.com/scala/scala/pull/5100